KarhooUISDK/CommonUI/LoyaltyInformation/LoyaltyInformationView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  LoyaltyInformationView.swift
+//  KarhooUISDK
+//
+//  Created by Aleksander Wedrychowski on 23/11/2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+/// View presenting user-friendly information how many poins shall be added/removed to/from Loyalty.
+struct KarhooLoyaltyInformationView: View {
+
+    private enum Constants {
+        static let iconSide: CGFloat = 16
+    }
+
+    // MARK: - Properties
+
+    let viewModel: ViewModel
+
+    // MARK: - Lifecycle
+
+    init(viewModel: ViewModel) {
+        self.viewModel = viewModel
+    }
+
+    // MARK: - Views
+
+    var body: some View {
+        HStack(spacing: UIConstants.Spacing.small) {
+            Image("kh_uisdk_info_icon", bundle: .current)
+                .resizable()
+                .renderingMode(.template)
```

Comment:
I always prefer to set this rendering mode in assets catalog. With this approach you don't need to add this modifier whenever icon is used.